src/fcu_io.cpp

```diff
@@ -435,30 +435,30 @@ void fcuIO::commandCallback(fcu_common::ExtendedCommand::ConstPtr msg)
   OFFBOARD_CONTROL_MODE mode = (OFFBOARD_CONTROL_MODE) msg->mode;
   OFFBOARD_CONTROL_IGNORE ignore = (OFFBOARD_CONTROL_IGNORE) msg->ignore;
 
-  int v1 = (int)(msg->value1 * 1000);
-  int v2 = (int)(msg->value2 * 1000);
-  int v3 = (int)(msg->value3 * 1000);
-  int v4 = (int)(msg->value4 * 1000);
+  double x = msg->x;
+  double y = msg->y;
+  double z = msg->z;
+  double F = msg->F;
 
   switch (mode)
   {
   case MODE_PASS_THROUGH:
-    v1 = saturate(v1, -1000, 1000);
-    v2 = saturate(v2, -1000, 1000);
-    v3 = saturate(v3, -1000, 1000);
-    v4 = saturate(v4, -1000, 1000);
+    x = saturate(x, -1.0, 1.0);
+    y = saturate(y, -1.0, 1.0);
+    z = saturate(z, -1.0, 1.0);
+    F = saturate(F, -1.0, 1.0);
```

Comment:
Should this saturate from -1 to 1 or from 0 to 1? I'm kind of forgetting, but is the purpose of this mode to pass straight servo commands to a fixed wing vehicle? If so then this would be throttle and should be 0 to 1?